secrets/secrets

```diff
@@ -19,3 +19,4 @@ packages.nuxeo.com-npm
 saucelabs-credentials
 saucelabs-js-client-credentials
 slack-cloud-support
+hxi-config
```

Comment:
Can you please sort it alphabetically?

Reply:
Made the required changes. Kindly re-check.